docs/solidity-to-cadence.mdx

```diff
@@ -0,0 +1,447 @@
+
+## Flow account model
+
+The [Flow account model](learn#concepts#accounts-and-keys) in Cadence combines storage for the keys and code
+(”smart contracts”) associated with an account with storage for the assets owned by that account. That’s right:
+In Cadence, your tokens are stored in your account, and not in a smart contract. Of course, smart contracts still
+define these assets and how they behave, but those assets can be securely stored in a user’s account through the
+magic of Resources.
+
+There is only one account type in Cadence also with an account address, similar to an Externally-Owned-Account
+(EOA) address in Ethereum. However, unlike Ethereum contract-accounts, accounts in Cadence also store contract code.
+Accounts realize ownership on Flow in being the container where keys, Resources, and contracts are stored on-chain.
+
+<Img src="https://storage.googleapis.com/flow-resources/documentation-assets/solidity-to-cadence/account-structure.png" />
+
+### PublicAccount and AuthAccount
+
+Fields, functions, Resources and/or Capabilities in a contract use [access modifiers](language@access-control)
+protecting privileged functionality or making it public. `PublicAccount` is the publicly available part of an account
+through which public functions or objects can be accessed by any account using the `getAccount` function.
+
+`AuthAccount` is available only to the signer of the transaction. An `AuthAccount` reference provides full access
+to that account's storage, public keys and contract code. Capabilities ensure that resources held in an account can be
+safely shared/accessed; access to `AuthAccount` is never given to other accounts.
+
+## Resources
+
+Resources are unique, [linear-types](https://en.wikipedia.org/wiki/Substructural_type_system#Linear_type_systems) which
+can never be copied or implicitly discarded, only moved between accounts. If, during development, a function fails to
+store a Resource obtained from an account in the function scope, semantic checks will flag an error. The run-time
+enforces the same strict rules in terms of allowed operations. Therefore, contract functions which do not properly
+handle Resources in scope before exiting will abort, reverting them to the original storage. These features of
+Resources make them perfect for representing tokens, both fungible and non-fungible. Ownership is tracked by where
+they are stored, and the assets can’t be duplicated or accidentally lost since the language itself enforces correctness.
+
+Flow encourages storing of data and compute on-chain and Resource-types makes this easier than even. Since Resources
+are always stored in accounts, any data that exists in Resource instances is seamlessly being managed on-chain
+without any explicit handling needed.
+
+## Capability-based access
+
+Remote access to stored objects can be delegated via [Capabilities](language#capability-based-access-control). This
```

Comment:
```suggestion
Remote access to stored objects can be managed via [Capabilities](language#capability-based-access-control). This
```

Small nitpick, you don't delegate to someone else the access everytime you use capabilities, you could use them also to restrict how you access your own resources